src/main/java/com/iota/iri/conf/SnapshotConfig.java

```diff
@@ -42,6 +77,13 @@
 
     interface Descriptions {
 
+        String LOCAL_SNAPSHOTS_ENABLED = "Flag that determines if local snapshots are enabled.";
+        String LOCAL_SNAPSHOTS_PRUNING_ENABLED = "Flag that determines if pruning of old data is enabled.";
+        String LOCAL_SNAPSHOTS_PRUNING_DELAY = "Only prune data that precedes the local snapshot by n milestones.";
+        String LOCAL_SNAPSHOTS_INTERVAL_SYNCED = "Take local snapshots every n milestones if the node is fully synced.";
+        String LOCAL_SNAPSHOTS_INTERVAL_UNSYNCED = "Take local snapshots every n milestones if the node is syncing.";
+        String LOCAL_SNAPSHOTS_DEPTH = "Number of milestones to keep.";
```

Comment:
Is this the number of filestones to keep prior to latest local snapshot? Can you elaborate the description a bit?

Reply:
Yep, that's clear; just to be more verbose in the doc.

Reply:
I agree that this description should be more informative.
what are the ranges, what do they mean for the users?

Reply:
Maybe also create a separate issue after merging so we can get these changes in first - its kind of tricky to break the whole shit down anyway